x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.js

```diff
@@ -170,19 +174,22 @@ export class AutoFollowPatternList extends PureComponent {
 
     if (!isAuthorized) {
       return (
-        <SectionUnauthorized
+        <PageError
           title={
             <FormattedMessage
               id="xpack.crossClusterReplication.autoFollowPatternList.permissionErrorTitle"
               defaultMessage="Permission error"
             />
           }
-        >
-          <FormattedMessage
-            id="xpack.crossClusterReplication.autoFollowPatternList.noPermissionText"
-            defaultMessage="You do not have permission to view or add auto-follow patterns."
-          />
-        </SectionUnauthorized>
+          error={{
+            error: (
```

Comment:
The `PageError` component doesn't seem to accept a custom error message and I really wanted to re-use it here. We can either do something like this, or modify the component to accept another prop